draft-ietf-tls-tls13.md

```diff
@@ -2746,13 +2711,18 @@ from the resumption master secret:
                             "resumption context", "", Hash.Length)
 ~~~~
 
-The client MAY use this PSK for future handshakes by including
-the ticket value in the "pre_shared_key" extension in its ClientHello
-({{pre-shared-key-extension}}) and supplying a suitable PSK cipher
-suite. Servers may send multiple tickets on a single connection, for
-instance after post-handshake authentication. For handshakes that
-do not use a resumption_psk, the resumption_context is a string of
-Hash.Length zeroes.
+The client MAY use this PSK for future handshakes by including the
+ticket value in the "pre_shared_key" extension in its ClientHello
+({{pre-shared-key-extension}}). Servers may send multiple tickets on a
```

Comment:
I'm having trouble reading this text. Can you rephrase?

On Mon, Jul 25, 2016 at 4:34 PM, Nick Sullivan <notifications@github.com>
wrote:

> In draft-ietf-tls-tls13.md
> <https://github.com/tlswg/tls13-spec/pull/559#discussion_r72074564>:
>
> > @@ -2746,13 +2711,18 @@ from the resumption master secret:
> >                              "resumption context", "", Hash.Length)
> >  ~~~~
> >
> > -The client MAY use this PSK for future handshakes by including
> > -the ticket value in the "pre_shared_key" extension in its ClientHello
> > -({{pre-shared-key-extension}}) and supplying a suitable PSK cipher
> > -suite. Servers may send multiple tickets on a single connection, for
> > -instance after post-handshake authentication. For handshakes that
> > -do not use a resumption_psk, the resumption_context is a string of
> > -Hash.Length zeroes.
> > +The client MAY use this PSK for future handshakes by including the
> > +ticket value in the "pre_shared_key" extension in its ClientHello
> > +({{pre-shared-key-extension}}). Servers may send multiple tickets on a
>
> Where are tickets used in post-handshake authentication? Multiple tickets
> are useful for unlinkability of simultaneous resumption requests, but not
> post-handshake auth.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tlswg/tls13-spec/pull/559/files/15126cf5a08c445aeed97c0c25c4f10c2c1b8f26#r72074564>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABD1oVWhxKjkcQUtImDwcORcKod9keQyks5qZMmEgaJpZM4JUEcw>
> .
>